barretenberg/cpp/src/barretenberg/vm2/simulation/events/contract_instance_retrieval_event.hpp

```diff
@@ -9,17 +9,16 @@ namespace bb::avm2::simulation {
  * Event emitted by the Contract Instance Manager for contract_instance_retrieval.pil gadget
  */
 struct ContractInstanceRetrievalEvent {
-    AztecAddress address;
-    ContractInstance contract_instance;
+    AztecAddress address = AztecAddress(0);
+    ContractInstance contract_instance{};
 
     // Tree context
-    FF nullifier_tree_root;
-    FF public_data_tree_root;
+    FF nullifier_tree_root = FF(0);
+    FF public_data_tree_root = FF(0);
 
     // Nullifier info
-    FF deployment_nullifier;
-    bool nullifier_exists;
-    AztecAddress deployer_protocol_contract_address;
```

Comment:
IMO it doesnt make sense to have this in the event when we expect all invocations to set this to `CONTRACT_INSTANCE_REGISTRY_CONTRACT_ADDRESS`. In fact the circuit is unprovable if this is not set.